### ISSUE TITLE
Allow opening recent projects (cmd+alt+o) with no active windows

### ISSUE THREAD

**Issue by porsager** [zed-industries/community#2261]
### Check for existing issues

- [X] Completed

### Describe the feature

It's currently only possible to open recent projects when there are no open windows. It would be great to be able to open this pane independently as it's currently necessary to do workarounds like, cmd+n to create a new untitled file, and then cmd+alt+o to get the recent projects pane.

**Comment by JosephTLyons** [comment]
I agree - this has been awkward to work with because of it. We should have a way to open recent projects, even if there is no window.
